src/main.cc

```diff
@@ -272,6 +281,8 @@ static int daemon_main(LiteClient& client, const bpo::variables_map& variables_m
 
   client.reportAktualizrConfiguration();
 
+  std::tuple<Hash, boost::uintmax_t, std::string> available_space_when_download_failed{Hash{"", ""}, 0, ""};
```

Comment:
trying to think through what happens here for a reboot. The path, `available_space_when_download_failed.2` won't be set, so the check for disk space will fail and we'll try and do an install. 

I *think* this is the right approach, just confirming i'm reading this right?

Reply:
Yes, that's correct. I am just not sure it's worth persisting the error state/size.

Reply:
i don't think so. just makes things complicated. Plus, reboots fix everything :)